lib/solidus_stripe/seeds.rb

```diff
@@ -0,0 +1,21 @@
+# frozen-string-literal: true
+
+module SolidusStripe
+  # Seed data to seamlessly integrate Solidus with Stripe.
+  module Seeds
+    # Default name for the `Spree::RefundReason` used for Stripe refunds.
+    #
+    # Changed via {SolidusStripe::Configuration#default_refund_reason_name}.
+    #
+    # @return [String]
+    DEFAULT_STRIPE_REFUND_REASON_NAME = "Stripe Refund"
```

Comment:
```suggestion
    DEFAULT_STRIPE_REFUND_REASON_NAME = "Refund generated from Stripe"
```

Personally, I prefer the reason text you used above, but not blocking.

Reply:
:ok_hand: changed!